client/blocks/get-apps/desktop-download-card.jsx

```diff
@@ -150,25 +160,33 @@ class DesktopDownloadCard extends Component {
 				return translate(
 					'{{firstAvailableLink}}{{firstAvailableIcon /}}Linux (.deb){{/firstAvailableLink}}' +
 						'{{secondAvailableLink}}{{secondAvailableIcon /}}Windows{{/secondAvailableLink}}' +
-						'{{thirdAvailableLink}}{{thirdAvailableIcon /}}Mac{{/thirdAvailableLink}}',
+						'{{thirdAvailableLink}}{{thirdAvailableIcon /}}Mac (Intel){{/thirdAvailableLink}}' +
+						'{{fourthAvailableLink}}{{fourthAvailableIcon /}}Mac (Apple Silicon){{/fourthAvailableLink}}',
 					{ components: this.getTranslateComponents( platform ) }
 				);
 			default:
 				return translate(
-					'{{firstAvailableLink}}{{firstAvailableIcon /}}MacOS{{/firstAvailableLink}}' +
-						'{{secondAvailableLink}}{{secondAvailableIcon /}}Linux (.tar.gz){{/secondAvailableLink}}' +
-						'{{thirdAvailableLink}}{{thirdAvailableIcon /}}Linux (.deb){{/thirdAvailableLink}}',
+					'{{firstAvailableLink}}{{firstAvailableIcon /}}MacOS (Intel){{/firstAvailableLink}}' +
+						'{{secondAvailableLink}}{{secondAvailableIcon /}}MacOS (Apple Silicon){{/secondAvailableLink}}' +
+						'{{thirdAvailableLink}}{{thirdAvailableIcon /}}Linux (.tar.gz){{/thirdAvailableLink}}' +
+						'{{fourthAvailableLink}}{{fourthAvailableIcon /}}Linux (.deb){{/fourthAvailableLink}}',
 					{ components: this.getTranslateComponents( platform ) }
 				);
 		}
 	}
 
 	getButtonClickHandler( platform ) {
-		const { trackWindowsClick, trackMacClick, trackLinuxTarClick, trackLinuxDebClick } = this.props;
+		const {
+			trackWindowsClick,
+			trackMacIntelClick,
+			trackMacSiliconClick,
+			trackLinuxTarClick,
+			trackLinuxDebClick,
+		} = this.props;
 
 		switch ( platform ) {
 			case 'MacIntel':
-				return trackMacClick;
+				return trackMacIntelClick && trackMacSiliconClick;
```

Comment:
We cannot "join" these `trackPlatformClick` with `&&`.
The result of `func1 && func2` would be a boolean value.
That would work on the command line, though.

Let's create a new case for `MacSilicon` even if that doesn't exist natively.

```suggestion
				return trackMacIntelClick;
		case 'MacSilicon':
				return trackMacSiliconClick;
```